src/main/java/org/prebid/server/spring/config/bidder/util/BidderDepsAssembler.java

```diff
@@ -156,23 +158,34 @@ private Bidder<?> bidder(CFG configProperties) {
     private void validateCapabilities(String alias, CFG aliasConfiguration, String coreBidder, CFG coreConfiguration) {
         final MetaInfo coreMetaInfo = coreConfiguration.getMetaInfo();
         final MetaInfo aliasMetaInfo = aliasConfiguration.getMetaInfo();
-        final List<MediaType> coreAppMediaTypes = coreMetaInfo.getAppMediaTypes();
-        final List<MediaType> coreSiteMediaTypes = coreMetaInfo.getSiteMediaTypes();
-        final List<MediaType> aliasAppMediaTypes = aliasMetaInfo.getAppMediaTypes();
-        final List<MediaType> aliasSiteMediaTypes = aliasMetaInfo.getSiteMediaTypes();
+
+        final Set<MediaType> coreAppMediaTypes = new HashSet<>(coreMetaInfo.getAppMediaTypes());
+        final Set<MediaType> coreSiteMediaTypes = new HashSet<>(coreMetaInfo.getSiteMediaTypes());
+        //it's a workaround in order not to update all the bidder config files at once
+        final Set<MediaType> coreDoohMediaTypes = new HashSet<>(
+                Optional.ofNullable(coreMetaInfo.getDoohMediaTypes()).orElse(List.of()));
```

Comment:
No need to optimize this code. It will be executed only on startup. Also it may produce NPE.

Reply:
it's done in terms of config compatibility, so it's not required to add dooh media types in all the config yaml files